src/mpd.c

```diff
@@ -3619,6 +3628,7 @@ mpd_command_disableoutput(struct evbuffer *evbuf, int argc, char **argv, char **
     }
 
   memset(&param, 0, sizeof(struct output_get_param));
+  param.curid = 0;
```

Comment:
Remove this, the memset above has already zeroed the struct (same comment for the below)